nebula49

jacobgkau opened the pull request:
This adds the external overview and assembly manual for the nebula49.

jacobgkau commented:
This will now also include a couple of minor fixes for the nebula19 and nebula36 sections.

When merging, please do not squash (use `Rebase and merge`, not `Squash and merge`).

leviport commented:
This review is kinda tricky since Github prevents me from commenting on lines that weren't changed in the PR. I guess I'll just do individual comments instead.

At the top of `src/models/nebula49-1/README.md` it looks like the External Overview and Parts & Assembly links are missing. Also, the name should be changed to `nebula49` to match 36 and 19
![2023-06-30_09-08-18](https://github.com/system76/tech-docs/assets/13512097/fb600354-cce5-4a68-9d65-f91a94eeb3cf)
![2023-06-30_09-07-58](https://github.com/system76/tech-docs/assets/13512097/c36a1ab3-4674-48d6-9789-804a19013fd1)

(still reviewing)

leviport commented:
Not sure where we landed with the optional second side intake fan, but if we're making that an option, then this note will need to be updated in `src/models/nebula49-1/assembly.md` on L55
![2023-06-30_10-42-10](https://github.com/system76/tech-docs/assets/13512097/04ff6d08-8805-4dfd-b46e-bcc0bfb4bd92)

(still reviewing)

leviport commented:
On L126 of `src/models/nebula49-1/assembly.md`, the link `[remove the CPU shroud](removing-the-cpu-shroud).` is missing a `#` so it 404's

(still reviewing)

leviport commented on src/models/nebula49-1/assembly.md:
```diff
@@ -220,7 +216,7 @@ These instructions use the Noctua `NH-U12S` as an example.
 ## Installing the GPU:
 
-nebula36 supports GPUs up to 318.80mm long. For optimal cooling and ease of installation, GPUs longer than 308.80mm are not recommended.
+nebula49 supports GPUs up to 318.80mm long. For optimal cooling and ease of installation, GPUs longer than 308.80mm are not recommended.
```
GPU lengths need to be updated

leviport commented on src/models/nebula49-1/assembly.md:
```diff
@@ -250,9 +246,9 @@ nebula36 is designed to work with an optional SATA backplane to allow for easy h
-![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+![2.5" SATA drive cage cover](./img/25-drive-covers.webp)
 
 3. Unscrew and remove the 2.5" drive cage.
```
Cage should be plural

jacobgkau replied:
Since we sell the SATA backplanes individually, someone might be installing only one. I don't want to have to repeat "for each backplane you're installing" on every single line of this section, so I was trying to make it written in singular and then just say "do the section for each one." I'll try to make it more clear.

leviport commented on src/models/nebula49-1/assembly.md:
```diff
@@ -318,17 +314,17 @@ The 140mm bottom case fan (`BQ SIW4-14025-LF-PWM`) is preinstalled, but can be r
 
```
Step 4 above should be updated like nebula36 (no acrylic spacer, plus the new wording)

leviport commented on src/models/nebula49-1/assembly.md:
```diff
@@ -318,17 +314,17 @@ The 140mm bottom case fan (`BQ SIW4-14025-LF-PWM`) is preinstalled, but can be r
 ## Installing side intake fans:
 
-nebula36 supports up to two 120mm side intake fans mounted to the GPU brace. One 120mm fan (`BQ SIW4-12025-MF-PWM`) is sold as an optional add-on at the time of purchase.
+nebula49 supports up to two 120mm side intake fans mounted to the GPU brace. One 120mm fan (`BQ SIW4-12025-MF-PWM`) is sold as an optional add-on at the time of purchase.
```
If two fans become an option, this sentence should be updated

leviport commented on src/models/nebula49-1/assembly.md:
```diff
@@ -337,10 +333,11 @@ nebula36 supports up to two 120mm side intake fans mounted to the GPU brace. One
 ![Mounted side fan](./img/side-fan-mounted.webp)
 
 5. Plug the side fan into one of the headers on the GPU brace's fan splitter board. The cearby velcro strap can be used to hold the fan's cable.
```
s/cearby/nearby/g

leviport reviewed (changes requested):
Review complete, sorry it's scattered across multiple comments.

jacobgkau commented:
Addressed all the feedback.

Regarding the side intake fans, I had tried to make it vague since we didn't know what the final decision was going to be on that (missed the line in the GPU brace section that you pointed out.) However, it looks like we now offer two or zero as the only options, so I've updated the copy to reflect that.

leviport reviewed (approved):
Looks great!
